CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+## 6.0.0 (May 28, 2020)
+[Full Changelog](https://github.com/vcr/vcr/compare/v5.1.0...v6.0.0)
+
+- [breaking] Require Ruby 2.3 or newer (#816)
+- [new] Add option to downcase cassette names before saving (#802)
+- [patch] Fix: Prevent storing empty `http_version` on cassettes (#709)
+- [patch] Fix tests with excon >= 0.63 (#809)
```

Comment:
FWIW, I think while this is a patch, it's not really user facing patch.

Reply:
I'll remove it.

The change in gemspec was about development deps anyway.

https://github.com/vcr/vcr/commit/b38369dcace01aa4026abd42209a3025ed719d62